pysqa/wrapper/torque.py

```diff
@@ -19,7 +19,7 @@
 class TorqueCommands(SchedulerCommands):
     @property
     def submit_job_command(self):
-        return ["qsub", "-terse"]
```

Comment:
I am a bit surprised about this one, as it appears in the documentation of the sun grid engine: https://gridscheduler.sourceforge.net/htmlman/manuals.html - can we clarify which systems or system version support this command?

Reply:
Here is some comparison for SGE vs. PBS https://softpanorama.org/HPC/PBS_and_derivatives/Reference/pbs_command_vs_sge_commands.shtml

Reply:
PBSPro - I can see that PBSPro is the closed-source version of the OpenPBS project, Torque is the continued fork of OpenPBS after it was abandoned.

Do we want to create a new set of templates specifically for PBSPro?

Reply:
at any rate there is no -terse option in torque's documentation, and neither in the PBSPro version:
```bash
(base) [hlm562@gadi-login-05 ~]$ qsub -H
qsub: invalid option -- 'H'
usage: qsub [-a date_time] [-A account_string] [-c interval]
        [-C directive_prefix] [-e path] [-f ] [-h ] [-I [-X]] [-j oe|eo] [-J X-Y[:Z]]
        [-k keep] [-l resource_list] [-m mail_options] [-M user_list]
        [-N jobname] [-o path] [-p priority] [-P project] [-q queue] [-r y|n]
        [-R o|e|oe] [-S path] [-u user_list] [-W otherattributes=value...]
        [-S path] [-u user_list] [-W otherattributes=value...]
        [-v variable_list] [-V ] [-z] [script | -- command [arg1 ...]]
       qsub --version
```

Reply:
Ok, can we have on version for torque and then derive from there a second version for the sun grid engine?

Reply:
I see we already have a separate implementation for the sun grid engine https://github.com/pyiron/pysqa/blob/main/pysqa/wrapper/sge.py